testing/tests/DevExpress.exporter/jspdfParts/jspdf.dataGrid.tests.js

```diff
@@ -21,23 +21,73 @@ QUnit.testStart(() => {
 let helper;
 
 const moduleConfig = {
+    jsPDFUnit: 'pt',
     beforeEach: function() {
         // The transpiling of the script on the drone and locally has differences that affect the imported jsPDF type.
         const _jsPDF = isFunction(jsPDF) ? jsPDF : jsPDF.jsPDF;
-        this.jsPDFDocument = _jsPDF('p', 'pt', 'a4');
+        this.jsPDFDocument = _jsPDF('p', this.jsPDFUnit, 'a4');
 
         helper = new JSPdfDataGridTestHelper(this.jsPDFDocument);
     }
 };
 
-QUnit.module('Scenarios, generate autoTable options', moduleConfig, () => {
+['mm', 'cm', 'em', 'px', 'pt'].forEach((unit) => {
+    const _moduleConfig = $.extend(moduleConfig, {
+        jsPDFUnit: unit
+    });
+
+    QUnit.module('Scenarios, check generated content', _moduleConfig, () => {
+        const getOptions = (context, dataGrid, expectedCustomizeCellArgs, options) => {
+            const { autoTableOptions = {} } = options || {};
+
+            const result = {
+                component: dataGrid,
+                jsPDFDocument: context.jsPDFDocument
+            };
+            result.autoTableOptions = autoTableOptions;
+            return result;
+        };
+
+        QUnit.test(`Table and column widths, jsPDFUnit: ${unit}`, function(assert) {
+            const done = assert.async();
+            const ds = [{ id: 1, name: 'test' }];
+            const dataGrid = $('#dataGrid').dxDataGrid({
+                dataSource: ds,
+                keyExpr: 'id',
+                columns: [
+                    { dataField: 'id', width: 50, caption: 'id' },
+                    { dataField: 'name', caption: 'name' }
+                ],
+                loadingTimeout: undefined,
+                showColumnHeaders: true,
+                width: 500
+            }).dxDataGrid('instance');
+            const expectedCells = [];
+            const options = {
+                autoTableOptions: {
+                    styles: { fillColor: [255, 0, 0] }
+                }
+            };
+
+            exportDataGrid(getOptions(this, dataGrid, expectedCells, options)).then((jsPDFDocument) => {
+                const autoTableOptions = jsPDFDocument.autoTable.__autoTableOptions;
+                helper.checkTableAndColumnWidths(500, [50, 'auto'], autoTableOptions);
+                helper.checkTableAndColumnWidthsInOutput(500, [50, 450]);
+                done();
+            });
+        });
+    });
+});
+
+QUnit.module('Scenarios, check autoTableOptions', moduleConfig, () => {
     const getOptions = (context, dataGrid, expectedCustomizeCellArgs, options) => {
```

Comment:
'expectedCustomizeCellArgs' is not used, remove?

Reply:
ok